jbmc/regression/jbmc-strings/IsEmptyConstantEvaluation/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+Main.class
+--function Main.constantIsEmpty
+^Generated [0-9]+ VCC\(s\), 0 remaining after simplification$
+^VERIFICATION SUCCESSFUL$
```

Comment:
We should also have tests for which the verification should fail

Reply:
Added